src/Orc.CsvTextEditor/Behaviors/RegisterCstTextEditorControlsServicesBehavior.cs

```diff
@@ -0,0 +1,85 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="RegisterCstTextEditorControlsServicesBehavior.cs" company="WildGums">
+//   Copyright (c) 2008 - 2016 WildGums. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace Orc.CsvTextEditor
+{
+    using System;
+    using System.ComponentModel;
+    using Catel.IoC;
+    using Catel.Windows.Interactivity;
+    using Services;
+
+    public class RegisterCstTextEditorControlsServicesBehavior : BehaviorBase<CsvTextEditorControl>
+    {
+        protected override void OnAttached()
```

Comment:
not done - it breaks the app - we should stick to OnAttached - cause we need to register services before contol is Loaded